library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -150,7 +151,14 @@ public void onCreate() {
 
         downloadClientReadyChecker = getDownloadClientReadyChecker();
 
-        mNotifier = new DownloadNotifier(this, getNotificationImageRetriever(), getResources());
+        NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        NotificationDisplayer notificationDisplayer = new NotificationDisplayer(
+                this,
+                notificationManager,
+                getNotificationImageRetriever(),
+                getResources());
+
```

Comment:
Oh I was about to change it now for you now, already merged. When I see it again I'll remove the line ;)